src/Microsoft.SqlTools.ServiceLayer/Profiler/ProfilerService.cs

```diff
@@ -120,6 +121,39 @@ public void InitializeService(ServiceHost serviceHost)
             this.SessionMonitor.AddSessionListener(this);
         }
 
+        /// <summary>
+        /// Handle request to start a profiling session
+        /// </summary>
+        internal async Task HandleStartProfilingRequest(StartProfilingParams parameters, RequestContext<StartProfilingResult> requestContext)
+        {
+            await Task.Run(async () =>
+            {
+                try
+                {
+                    ConnectionInfo connInfo;
+                    ConnectionServiceInstance.TryFindConnection(
+                        parameters.OwnerUri,
+                        out connInfo);
+                    if (connInfo != null)
+                    {
+                        CancellationTokenSource threadCancellationToken = new CancellationTokenSource();
+                        // create a new XEvent session and Profiler session
+                        var xeSession = this.XEventSessionFactory.GetXEventSession(parameters.SessionName, connInfo);
+                        // //Create XELiveEvsentStream here, using ConnectionInfo Strings.
+                        var connectionString = ConnectionService.BuildConnectionString(connInfo.ConnectionDetails, true);
+                        // //need to start session otherwise XLiveEventStreamer won't work.
```

Comment:
looks like there are double comment indicators `//`.  nit: not really sure saying we need to start the session or else streaming won't work.  That seems self-explanatory, IMO.